src/radical/pilot/pmgr/launching/default.py

```diff
@@ -15,6 +15,8 @@
 import radical.saga            as rs
 import radical.utils           as ru
 
+rsfs = rs.filesystem
```

Comment:
this could be removed, since it was used in removed methods `_handle_pilot_input_staging` and `_handle_pilot_output_staging`

Reply:
@andre-merzky can you please look here? (that were initial changes in this branch)

Reply:
Ack,   that can be removed as staging moved out of here.